proto/ethermint/evm/v1/evm.proto

```diff
@@ -240,3 +240,43 @@ message TraceConfig {
   // tracer_json_config configures the tracer using a JSON string
   string tracer_json_config = 13 [(gogoproto.jsontag) = "tracerConfig"];
 }
+
+// VirtualFrontierContract are virtual contracts that able to call directly without actual invoking EVM.
+// Prohibited to call directly from EVM.
+message VirtualFrontierContract {
+  // address is the contract address
+  string address = 1;
+  // active indicate the activation of the contract. If not active, invoking methods is disabled.
+  bool active = 2;
+  // type of the virtual frontier contract
+  uint32 type = 3;
+  // metadata contains marshalled binary of the metadata of the virtual frontier contract
+  bytes metadata = 4;
+}
+
+// VFBankContractMetadata contains metadata of the Bank Contract, which represents ERC-20 contracts as proxy of the user bank assets.
+// Each bank contract is represented as a single asset of user bank.
+message VFBankContractMetadata {
+  // min_denom is the base denomination of the asset
+  string min_denom = 1;
+}
+
+// UpdateVirtualFrontierBankContractsProposal is a gov Content type to update the virtual frontier bank contracts.
```

Comment:
wdym by `virtual frontier bank contracts`? what the difference with `VirtualFrontierContract`?

Reply:
`VFC` is abstraction class, `VFBC` inherits the `VFC`. In the future, we can have smt like `virtual frontier staking contract` which also a VFC